processor/attributesprocessor/attributes_log_test.go

```diff
@@ -76,7 +76,7 @@ func TestLogProcessor_NilEmptyData(t *testing.T) {
 	factory := NewFactory()
 	cfg := factory.CreateDefaultConfig()
 	oCfg := cfg.(*Config)
-	oCfg.Settings.Actions = []attraction.ActionKeyValue{
+	oCfg.Actions = []attraction.ActionKeyValue{
```

Comment:
What rule is enforcing to remove the embedded struct? Is this not potentially an issue if we have multiple embedded structs with the same field down the road?

Reply:
https://go.dev/play/p/2vLJH1U8Jjz shows a simple example. If you have ambiguity, Golang stops being able to compile. I think the rule is QF1008 right?

Reply:
I am ok with this but I remember that a while back, we had instructions to make the embedded struct explicit.
What tends to win is what linters say, since they are enforceable, rather than a soft code preference.